lib/transform.js

```diff
@@ -240,6 +240,7 @@ TreeTransformer.prototype = new TreeWalker;
     });
 
     _(AST_Export, function(self, tw){
+        if (self.exported_definition) self.exported_definition = self.exported_definition.transform(tw);
```

Comment:
I took the liberty to assume you meant `exported_definition` in all three instances here.